EIPS/eip-225.md

```diff
@@ -0,0 +1,474 @@
+---
+eip: 225
+title: Clique proof-of-authority consensus protocol
+author: Péter Szilágyi <peter@ethereum.org>
```

Comment:
I'm not sure you will be able to accept any later changes with this email address.

Reply:
The @eip-automerger checks the author field and verifies this against the GitHub account adding a PR to update or accept changes to the Draft in a PR. Since the e-mail address on your GitHub account is different, it won't automatically accept your changes in the future. You can also decide to add your @karalabe account name instead of e-mail address for this to work.

Reply:
Done.